### DISCUSSION TITLE
When batch correction cannot be effectively performed, how should it be handled?

### DISCUSSION THREAD

**Discussion by Dragonlongzhilin** [stuart-lab/signac#1728] [Q&A]
Thank you for providing such a useful tool. When using Harmony for batch correction, I noticed that tumor cells in different samples show distinct separation, while normal cells cluster by cell type. Even when I apply Harmony separately to tumor cells, they still do not aggregate.
I'm uncertain whether this indicates significant differences in chromatin accessibility among tumor cells across samples or if it's due to issues with my processing methods. 
In this situation, do you have any suggestions?

**Comment by timoast** [comment]
It is fairly common for cancer samples that the tumor cells will be very much donor-specific, while the non-cancer cells can group together more easily. Usually your goal here is to annotate the cell types, and you should be able to do that even if the cancer cells do not cluster together. As long as you've processed the data the same for the different samples (mapped to the same genome assembly using the same pipeline), and quantified the same set of peaks across all the samples, it's unlikely to be a data processing issue.
